templates/contribute/index.md

```diff
@@ -45,11 +109,17 @@ After responding appropriately to bors (if necessary), the PR will get added to
 
 Here are some other frequently-used labels:
 
-- A **"WIP"** (= work in progress) PR still needs some foundational work (e.g. maybe it still contains `sorry`s) before getting reviewed. Post a WIP if you want to announce that you're working on something you expect to finish soon. 
+- A **"WIP"** (= work in progress) PR still needs some foundational work (e.g. maybe it still contains `sorry`s) before getting reviewed. Post a WIP if you want to announce that you're working on something you expect to finish soon.
+
+- A **"RFC"** (= request for comment) is a PR about a change that might be controversial or need a decision from an expert about
+whether to proceed at all.
+
+- You can add **"awaiting-CI"**, which will temporarily hide the PR on the main review queue.
+  This label will be automatically removed when CI is complete.
 
 - Consider adding the **"help wanted"** label to directly solicit contributions.
 
-- The **"blocked-by-other-PR"** label means that some specific other PR(s) should be resolved before addressing this one. If you add the "blocked-by-other-PR" label to your PR, please include the PR numbers of the dependencies in the title and PR comment so that others can see at a glance which PRs should be reviewed first.
+- The **"blocked-by-other-PR"** label means that some specific other PR(s) should be resolved before addressing this one. If you add the "blocked-by-other-PR" label to your PR, please include the PR numbers of the dependencies in the PR comment so that others can see at a glance which PRs should be reviewed first.
```

Comment:
```suggestion
- The **"blocked-by-other-PR"** label means that some specific other PR(s) should be resolved before addressing this one. To add the "blocked-by-other-PR" label to your PR, include the PR numbers of the dependencies in the PR comment (following the example hidden in the comment there) so that others can see at a glance which PRs should be reviewed first. The label will be added automatically by a bot and will also be removed automatically when the other PRs have been merged.
```